manifests/repo/debian.pp

```diff
@@ -67,7 +67,13 @@
     }
   }
 
-  if ($sury and $php::globals::php_version in ['5.6','7.1','7.2'] ) {
```

Comment:
would be better to simply add the supported php version of sury in the array, only this way non valid php versions could be catched.
with your change a php version like `8.0` would be allowed

Reply:
Do you really think that hardcoded list of supported php versions is a good idea?

Reply:
not really, i prefer to have them configureable via hiera with sane default values directly set in this module with hiera. i would prefer that for many hardcoded values (like the repo key id) but without a bigger refactoring this is not possible.

to simply remove the version check feels wrong to me.